flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/TestingJobMasterPartitionTracker.java

```diff
@@ -111,13 +106,14 @@ public void stopTrackingAndReleasePartitions(Collection<ResultPartitionID> resul
     }
 
     @Override
-    public void stopTrackingAndReleasePartitionsFor(ResourceID producingTaskExecutorId) {
-        stopTrackingAndReleaseAllPartitionsConsumer.accept(producingTaskExecutorId);
+    public void stopTrackingAndReleaseOrPromotePartitions(
+            Collection<ResultPartitionID> resultPartitionIds) {
+        stopTrackingAndReleaseOrPromotePartitionsConsumer.accept(resultPartitionIds);
     }
 
     @Override
-    public void stopTrackingAndReleaseOrPromotePartitionsFor(ResourceID producingTaskExecutorId) {
-        stopTrackingAndReleaseOrPromotePartitionsConsumer.accept(producingTaskExecutorId);
+    public Collection<ResultPartitionDeploymentDescriptor> getAllTrackedPartitions() {
+        return Collections.emptyList();
```

Comment:
maybe also add a `Function` for this method to make it customizable?

Reply:
ok